app/views/chargeback_rate/_edit.html.haml

```diff
@@ -0,0 +1,72 @@
+- url = url_for_only_path(:action => 'cb_rate_form_field_changed', :id => @edit[:rec_id] || "new")
+- currency = Currency.currencies_for_select
+= render :partial => "layouts/flash_msg"
+#form_div
+  %h3
+    = _('Basic Info')
+  .form-horizontal
+    .form-group
+      %label.col-md-2.control-label
+        = _('Description')
+      .col-md-8
+        = text_field_tag("description", @edit[:new][:description],
+                        :maxlength => ViewHelper::MAX_NAME_LEN, "data-miq_observe" => {:interval => '.5', :url => url}.to_json,
+                        :class => "form-control")
+        = javascript_tag(javascript_focus('description'))
+    .form-group
+      %label.col-md-2.control-label
+        = _('Type')
+      .col-md-8
+        - unless @edit[:rec_id]
+          = select_tag("rate_type",
+            options_for_select(["Compute", "Storage"], @edit[:new][:rate_type]),
+            "class" => "selectpicker")
+          :javascript
+            miqInitSelectPicker();
+            miqSelectPickerEvent("rate_type", "#{url}");
+        - else
+          = _(@edit[:new][:rate_type])
```

Comment:
I'm wondering if the `_` does anything here and if it's safe to just print this out without any sanitization. 

cc @mzazrivec